lib/routes/token.js

```diff
@@ -249,6 +251,20 @@ function confirmRefreshToken(params) {
       throw AppError.invalidScopes();
     }
     tokObj.scope = params.scope;
+
+    var now = new Date();
+    var lastUsedAt = tokObj.lastUsedAt;
+
+    if ((now - lastUsedAt ) > REFRESH_LAST_USED_AT_UPDATE_AFTER_MS){
+      db.usedRefreshToken(encrypt.hash(params.refresh_token)).then(function() {
```

Comment:
@larissagaulia you already `.hash` the token in the mysql db method i think, is this doing it twice?